src/main/java/com/danplay/server/user/domain/entity/PreferSport.java

```diff
@@ -0,0 +1,29 @@
+package com.danplay.server.user.domain.entity;
+
+import com.danplay.server.user.domain.enumerations.Prefer;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Builder
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class PreferSport {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "preferSport_id")
```

Comment:
```suggestion
    @Column(name = "prefer_sport_id")
```
이 스네이크케이스에 더 적절할 것 같네요!